reactor-netty-http/src/test/java/reactor/netty/http/HttpCompressionClientServerTests.java

```diff
@@ -450,34 +439,26 @@ void testIssue825_1() {
 
 		AtomicReference<Throwable> error = new AtomicReference<>();
 		DisposableServer server1 =
-				HttpServer.create()
-				          .port(port1)
+				createServer(port1)
```

Comment:
note: this test doesn't dispose `server1` and `server2` in case of any assertion error.

Reply:
fixed